### PR TITLE
posix: Do not take disk offline on I/O errors

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
posix: Do not take disk offline on I/O errors

## Motivation and Context
Choosing maxAllowedIOError is arbitrary and
prone to errors, when drives might be perfectly
capable of taking I/O with only few locations
return I/O error. This is a hindrance of sort
where backend filesystems like ZFS can automatically
fix and handle these scenarios.
    
The added problem with current approach that we
take the drive offline, making it virtually impossible
to bring it online without restart the server which
is not desirable on a busy cluster. Remove this state
such that let the backend return error appropriately
to the caller and let the caller decide what to do with
the error.

## How to test this PR?
You need to use filesystems like zfs or btrfs

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [x] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-worm.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#7eaeda44-e579-429d-8883-accfb78df1d4) |
| mint-large-bucket.sh | :x: [more...](#df1c4bd3-904b-4559-abb1-4ab7659ef879) |
| mint-fs.sh | :x: [more...](#06506b03-daf7-4089-b86e-fc3d6b45a2d9) |
| mint-dist-xl.sh | :x: [more...](#2b609f80-e2e6-457e-a436-02a6dcd6e6a8) |
| mint-gateway-nas.sh | :x: [more...](#03d00233-18b8-427a-a060-b485a91c7d8b) |
<a name="2b609f80-e2e6-457e-a436-02a6dcd6e6a8"></a>
### 8836-73e2a50/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31163
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp ac3f532ff2f5:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 2 minutes and 22 seconds
{
  "name": "awscli",
  "duration": 4024,
  "function": "aws --endpoint-url http://72.28.97.51:31163 s3api head-object --bucket awscli-mint-test-bucket-14564 --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "awscli-mint-test-bucket-14564"
}

Executed 0 out of 13 tests successfully.
```
<a name="df1c4bd3-904b-4559-abb1-4ab7659ef879"></a>
### 8836-73e2a50/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.59:30043
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e6a8e615fe5c:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 2 minutes and 18 seconds
{
  "name": "awscli",
  "duration": 4104,
  "function": "aws --endpoint-url http://72.28.97.59:30043 s3api head-object --bucket awscli-mint-test-bucket-3114 --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "awscli-mint-test-bucket-3114"
}

Executed 0 out of 13 tests successfully.
```
<a name="03d00233-18b8-427a-a060-b485a91c7d8b"></a>
### 8836-73e2a50/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32335
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e92724126209:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 1 minutes and 9 seconds
{
  "name": "awscli",
  "duration": 3960,
  "function": "aws --endpoint-url http://72.28.97.56:32335 s3api head-object --bucket awscli-mint-test-bucket-10656 --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "awscli-mint-test-bucket-10656"
}

Executed 0 out of 13 tests successfully.
```
<a name="7eaeda44-e579-429d-8883-accfb78df1d4"></a>
### 8836-73e2a50/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:30722
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp f83ce334f11e:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 12 seconds
{"name": "awscli", "duration": 309148, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:30722/awscli-mint-test-bucket-4979""}

Executed 0 out of 13 tests successfully.
```
<a name="06506b03-daf7-4089-b86e-fc3d6b45a2d9"></a>
### 8836-73e2a50/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32005
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a8a0322d396d:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 2 minutes and 8 seconds
{
  "name": "awscli",
  "duration": 3951,
  "function": "aws --endpoint-url http://72.28.97.61:32005 s3api head-object --bucket awscli-mint-test-bucket-25594 --key datafile-1-kB\n",
  "status": "FAIL",
  "error": "awscli-mint-test-bucket-25594"
}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#7ee0ec6c-ec24-42d2-981d-2811c0e9c20a) |
| mint-large-bucket.sh | :x: [more...](#61532614-8a96-46c3-8caa-8e444679b16f) |
<a name="7ee0ec6c-ec24-42d2-981d-2811c0e9c20a"></a>
### 8836-c74344e/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31133
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 2c5d58e2c6dc:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 15 seconds
{
  "name": "awscli",
  "duration": 12896,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="61532614-8a96-46c3-8caa-8e444679b16f"></a>
### 8836-c74344e/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:30656
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 58a2c8b9d996:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 1 minutes and 29 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 4 seconds
(4/13) Running mc tests ... done in 42 seconds
(5/13) Running minio-go tests ... FAILED in 2 minutes and 59 seconds
{
  "args": {},
  "duration": 66551,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#5250929b-e059-48de-9f39-0d26252ee894) |
<a name="5250929b-e059-48de-9f39-0d26252ee894"></a>
### 8836-fa09d0b/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31536
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 253abe9b9f2c:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 22 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 7 seconds
(4/13) Running mc tests ... done in 1 minutes and 4 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 46 seconds
{
  "args": {},
  "duration": 110048,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#1bd87de3-8791-43ab-a472-a3baa18bd5c6) |
<a name="1bd87de3-8791-43ab-a472-a3baa18bd5c6"></a>
### 8836-045255b/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:30088
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp aa7c5b606d8b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 22 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 1 minutes and 4 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 40 seconds
{
  "args": {},
  "duration": 106881,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by vadmeste** [comment]
LGTM, but also it is probably good to keep the io errors count that the server encounters, just to export the information another day

**Comment by harshavardhana** [comment]
@vadmeste it is not really meaningful we are not making use of any such information and have no real plans to do so in future. Unless we have some Prometheus metrics to calculate that.

I would just keep it barebones for now and bring it back as and when we need it.

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#abffb661-629c-45cf-9b1f-e1089b2bedaa) |
<a name="abffb661-629c-45cf-9b1f-e1089b2bedaa"></a>
### 8836-93ae438/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:32360
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 40151e306455:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 12 seconds
{"name": "awscli", "duration": 9938, "function": "make_bucket\n", "status": "FAIL", "error": "Could not connect to the endpoint URL: "http://72.28.97.56:32360/awscli-mint-test-bucket-9287""}

Executed 0 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by krishnasrinivas — APPROVED** [review]
ioErrCount is unused now, should be removed.

> LGTM, but also it is probably good to keep the io errors count that the server encounters, just to export the information another day

I would also prefer to keep this count and export in prometheus otherwise we would assume a bug in minio and be misled.

**Comment by harshavardhana** [comment]
For that we need a comprehensive change @krishnasrinivas to bring Prometheus at disk level. For now I will leave this as is..

File: cmd/posix.go [hunk 1]
```diff
@@ -86,8 +85,8 @@ type posix struct {
 	totalUsed  uint64 // ref: https://golang.org/pkg/sync/atomic/#pkg-note-BUG
 	ioErrCount int32  // ref: https://golang.org/pkg/sync/atomic/#pkg-note-BUG
```
Comment (harshavardhana) [inline cmd/posix.go]:
```suggestion
```

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#6408663c-3428-41ed-abdb-a03d56edfb01) |
| mint-large-bucket.sh | :x: [more...](#532071aa-e51c-48b5-a15e-6fb71b8a18af) |
| mint-worm.sh | :x: [more...](#adc6e8c3-f78f-439d-b172-31fb77514df9) |
| mint-gateway-nas.sh | :x: [more...](#76ed2023-cdc9-4623-891a-8ebb3689a264) |
<a name="76ed2023-cdc9-4623-891a-8ebb3689a264"></a>
### 8836-1fb8243/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:31041
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0c5c91a1b61f:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 5 minutes and 14 seconds
{"name": "awscli", "duration": 312198, "function": "make_bucket\n", "status": "FAIL", "error": "Connect timeout on endpoint URL: "http://72.28.97.55:31041/awscli-mint-test-bucket-24311""}

Executed 0 out of 13 tests successfully.
```
<a name="532071aa-e51c-48b5-a15e-6fb71b8a18af"></a>
### 8836-1fb8243/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.58:31234
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp abae56e40de4:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 19 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 5 seconds
(4/13) Running mc tests ... done in 51 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 41 seconds
{
  "args": {},
  "duration": 105801,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
<a name="6408663c-3428-41ed-abdb-a03d56edfb01"></a>
### 8836-1fb8243/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:31277
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 34d39e98960b:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 12 seconds
{
  "name": "awscli",
  "duration": 9873,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="adc6e8c3-f78f-439d-b172-31fb77514df9"></a>
### 8836-1fb8243/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.55:32582
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 138dcbff6aee:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 2 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-aynke9vm91l6upuw9i817njllu48ik7davd7ua98tgczov2bnj3uza1",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 120360,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.55:32582/bucket-aynke9vm91l6upuw9i817njllu48ik7davd7ua98tgczov2bnj3uza1: dial tcp 72.28.97.55:32582: i/o timeout",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by kannappanr** [comment]
@harshavardhana PTAL Build failures

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#cb3fe87a-b76c-450b-bdea-ace9983b3b8d) |
<a name="cb3fe87a-b76c-450b-bdea-ace9983b3b8d"></a>
### 8836-946da7f/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:30860
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 0a9421a17f3a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 24 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 1 minutes and 13 seconds
(5/13) Running minio-go tests ... FAILED in 3 minutes and 35 seconds
{
  "args": {},
  "duration": 109448,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#8b9d8784-9779-4e4b-85b1-bc64db72795b) |
| mint-large-bucket.sh | :x: [more...](#1178c5a1-634d-401d-abce-1f73ce625315) |
<a name="8b9d8784-9779-4e4b-85b1-bc64db72795b"></a>
### 8836-1ae302e/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.56:30319
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp d246eb96c97e:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... FAILED in 8 seconds
{
  "name": "awscli",
  "duration": 5671,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (XMinioServerNotInitialized) when calling the CreateBucket operation (reached max retries: 4): Server not initialized, please try again."
}

Executed 0 out of 13 tests successfully.
```
<a name="1178c5a1-634d-401d-abce-1f73ce625315"></a>
### 8836-1ae302e/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.52:31005
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp bd8e256b885a:/mint/log /tmp/mint-logs'

(1/13) Running awscli tests ... done in 2 minutes and 19 seconds
(2/13) Running aws-sdk-java tests ... done in 1 seconds
(3/13) Running aws-sdk-ruby tests ... done in 6 seconds
(4/13) Running mc tests ... done in 54 seconds
(5/13) Running minio-go tests ... FAILED in 2 minutes and 46 seconds
{
  "args": {},
  "duration": 67646,
  "error": "Please reduce your request",
  "function": "testStorageClassMetadataCopyObject()",
  "message": "CopyObject failed on RRS",
  "name": "minio-go: testStorageClassMetadataCopyObject",
  "status": "FAIL"
}

Executed 4 out of 13 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**